app/views/layouts/platoniq/_mini_footer.html.erb

```diff
@@ -14,23 +14,23 @@
       <a rel="license" class="cc-badge"
          href="http://creativecommons.org/licenses/by-sa/4.0/"
          target="_blank" rel="noopener">
-        <%= image_tag("decidim/cc-badge.png", alt: "Creative Commons License" ) %>
+        <%= image_pack_tag("media/images/cc-badge.png", alt: "Creative Commons License" ) %>
       </a>
       <%= t("platoniq.layouts.footer.made_by_platoniq").html_safe %>
     </div>
     <div class="medium-6 large-4 column">
       <% if current_organization.official_img_footer? %>
         <div class="decidim-logo">
         <%= link_to  current_organization.official_url do %>
-          <%= image_tag current_organization.official_img_footer.url.to_s , alt: current_organization.name %>
+          <%= image_pack_tag current_organization.official_img_footer.url.to_s , alt: current_organization.name %>
```

Comment:
```suggestion
<% if current_organization.official_img_header.attached? %>
        <%= image_tag current_organization.attached_uploader(:official_img_header).path, class: "minimap-official_img_header" %>
      <% else %>
        <span class="float-right"><%= t("official_img_header", scope: "activemodel.attributes.organization") %> &gt; </span>
        <%= content_tag :div, nil, { class: "minimap-official_img_header placeholder float-right", style: "width: #{image_width(current_organization, :official_img_header) * size_factor }px; height: #{image_height(current_organization, :official_img_header) * size_factor }px;" } %>
      <% end %>
```

image_pack_tag can only be used for static assets. Check this example https://github.com/decidim/decidim/blob/c9ed6740e8025fd482b49dad71cc1788b5e572c0/decidim-admin/app/views/decidim/admin/organization_appearance/form/_minimap.html.erb